crates/teloxide-core/src/serde_multipart/unserializers/input_file.rs

```diff
@@ -101,7 +101,7 @@ impl Serializer for InputFileUnserializer {
         Ok(self)
     }
 
-    forward_to_unsuported_ty! {
+    forward_to_unsupported_ty! {
```

Comment:
This file seems to be lingering. There is no `mod` declaration that includes it. Should I remove it?

Reply:
Yes, please remove this and `unserializers.rs`, oops

Reply:
Removed ✔️